src/components/index.js

```diff
@@ -0,0 +1,3 @@
+import { OuterContainer, InnerContainer, LinksContainer } from "./landingPageComponents/";
+
+export { OuterContainer, InnerContainer, LinksContainer };
```

Comment:
Instead of importing and exporting from the `landingPageComponents` folder you can directly write like this:
```js
export * from "./landingPageComponents/";
```